### ISSUE TITLE
[Bug]: Confluence Kafka and AVRO format

### ISSUE THREAD

**Issue by george0st** [mlrun/mlrun#2649]
### MLRun Version checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the latest version of the MLRun Kit.

### Reproducible Example

```python
See attached jupyter notebook with fstore.deploy_ingestion_service 

BTW: I removed pwd from jupyter
```

### Issue Description

This error (from log) is generated based on AVRO format from Confluence Kafka:

{"level":"error","time":"2022-11-23T09:57:19.437Z","name":"processor.kafka-cluster.w0.python.logger","message":"Exception caught in handler","more":{"exc":"'utf-32-be' codec can't decode bytes in position 4-7: code point not in range(0x110000)","traceback":"Traceback (most recent call last):\n  File \"/opt/nuclio/_nuclio_wrapper.py\", line 122, in serve_requests\n    await self._handle_event(event)\n  File \"/opt/nuclio/_nuclio_wrapper.py\", line 317, in _handle_event\n    entrypoint_output = self._entrypoint(self._context, event)\n  File \"/opt/nuclio/main.py\", line 7, in handler\n    return context.mlrun_handler(context, event)\n  File \"/usr/local/lib/python3.7/site-packages/mlrun/serving/server.py\", line 332, in v2_serving_handler\n    return context._server.run(event, context, get_body)\n  File \"/usr/local/lib/python3.7/site-packages/mlrun/serving/server.py\", line 252, in run\n    body = json.loads(event.body)\n  File \"/usr/local/lib/python3.7/json/__init__.py\", line 343, in loads\n    s = s.decode(detect_encoding(s), 'surrogatepass')\n  File \"/usr/local/lib/python3.7/encodings/utf_32_be.py\", line 11, in decode\n    return codecs.utf_32_be_decode(input, errors, True)\nUnicodeDecodeError: 'utf-32-be' codec can't decode bytes in position 4-7: code point not in range(0x110000)\n","worker_id":"0"}}

[conf-kafka-ingest_ingest_service (2).ipynb.txt](https://github.com/mlrun/mlrun/files/10146845/conf-kafka-ingest_ingest_service.2.ipynb.txt)

### Expected Behavior

Support not only JSON, but also AVRO and ingestion (via fstore.ingest or fstore.deploy_ingestion_service) will be without error.

### Python Version

3.7.6

### MLRun Version

1.2.0-rc13

**Comment by assaf758** [comment]
A PR is in the works - will include fixing of the serving server to allow avro encoded kafka records, as well as end-to-end with example of the required graph step for decoding records.

**Comment by assaf758** [comment]
resolved in v1.2.1-rc2
